tecken/tests/test_download.py

```diff
@@ -358,20 +360,30 @@ def test_get_code_id_lookup(client, db, s3_helper):
         code_id=code_id,
     )
 
+    # Try normal download API url
     url = reverse(
         "download:download_symbol",
-        args=(debug_filename, views.BOGUS_DEBUG_ID, sym_file),
+        args=(debug_filename, debug_id, sym_file),
     )
+    response = client.get(url)
+    assert response.status_code == 302
 
-    # Try with no querystring params
+    # Try with code_file/code_id
+    url = reverse(
+        "download:download_symbol",
+        args=(code_file, code_id, sym_file),
+    )
     response = client.get(url)
-    assert response.status_code == 404
+    assert response.status_code == 302
+    parsed = urlparse(response["location"])
+    assert parsed.path == f"/{debug_filename}/{debug_id}/{sym_file}"
 
     # Try with querystring params
     response = client.get(url, {"code_file": code_file, "code_id": code_id})
     assert response.status_code == 302
     parsed = urlparse(response["location"])
     assert parsed.path == f"/{debug_filename}/{debug_id}/{sym_file}"
+    assert parsed.query == f"code_file={code_file}&code_id={code_id}"
```

Comment:
Mmm... I don't know if this might occasionally fail or not. It would depend on whether the transformations are stable. I should probably assume it's not stable and add the other possibility.